kafka-connect-neo4j/src/main/kotlin/streams/kafka/connect/common/ConfigurationMigrator.kt

```diff
@@ -64,7 +64,7 @@ class ConfigurationMigrator(private val settings: Map<String, String>) {
         SERVER_URI to PropertyConverter("neo4j.uri") { settings[SERVER_URI] as String },
         AUTHENTICATION_TYPE to PropertyConverter("neo4j.authentication.type") { settings[AUTHENTICATION_TYPE] as String },
         AUTHENTICATION_BASIC_USERNAME to PropertyConverter("neo4j.authentication.basic.username") {settings[AUTHENTICATION_BASIC_USERNAME] as String},
-        AUTHENTICATION_BASIC_PASSWORD to PropertyConverter("neo4j.authentication.basic.password") {settings[AUTHENTICATION_BASIC_PASSWORD] as String},
+        AUTHENTICATION_BASIC_PASSWORD to PropertyConverter("neo4j.authentication.basic.password") {""},
```

Comment:
maybe `***`?

Reply:
I originally thought `***` as well, but i'm assuming that the error message we would get for an empty password rather than incorrect password would be more informative to the user that it hasn't been filled.
I'm hoping it would be something like 'No password found' vs  'invalid password'

Reply:
ok, that also makes sense. let's keep it as empty as you suggested.